pymbar/tests/test_exp.py

```diff
@@ -4,7 +4,7 @@
 
 import numpy as np
 import pytest
-from pymbar import exp as pyexp
+from pymbar import other_estimators as other
```

Comment:
I would probably do this as `import other_estimators as estimators`, since bar is an estimator, not an `other`.  Will make usage clearer.